packages/core/src/components/time-picker-rework/time-picker-rework.scss

```diff
@@ -0,0 +1,119 @@
+/*
+ * SPDX-FileCopyrightText: 2023 Siemens AG
+ *
+ * SPDX-License-Identifier: MIT
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+@import 'common-variables';
+@import 'mixins/shadow-dom/component';
+@import 'components/form/input';
+
+:host {
+  @include ix-component;
+
+  @import 'legacy/components/forms';
+
+  @media (max-width: 576px) {
+    .clock {
+      padding: 1.5rem 0;
+    }
+
+    .header {
+      height: $medium-control-height !important;
+    }
+  }
+
+  @media (min-width: 576px) {
+    .clock {
+      padding: 4.25rem 0;
+    }
+
+    .header {
+      height: $large-control-height !important;
+    }
+  }
+
+  .container {
+    max-width: 352px;
+  }
```

Comment:
The container does not fit with the host in this case you cannot control the size of the component.
```
<Host>
        <div class="container">
```

If your try to resize the host component you get only empty space:
![image](https://github.com/siemens/ix/assets/2144313/4a139fe5-6830-4db0-a70f-9bbebd93e1fc)

If i check the implementation the additional `<div class="container"` can be replaced by the `<Host>`/`:host`

Reply:
resolved